### PR TITLE
Replace Russian version with nondead analog

### PR DESCRIPTION
**Pull request by Sanceilaks**

### CODE REVIEW

**Review by bnjbvr — CHANGES_REQUESTED** [review]
There are a bit too many references to gods for my taste in the README file, and a bit too many insults in the code. Can you change that?

**Comment by Sanceilaks** [comment]
No problems, 10 minutes

**Comment by Sanceilaks** [comment]
> There are a bit too many references to gods for my taste in the README file, and a bit too many insults in the code. Can you change that?

As for "God", this is an old Russian meme "we are Russians, God is with us!". This is not meant to belittle or insult others, just a joke.
Insults removed

File: README.md [hunk 1]
```diff
@@ -99,7 +99,7 @@ Please don't introduce swear words, though: we will not excuse your French.
 - German: [rost](https://github.com/michidk/rost)
 - Polish: [rdza](https://github.com/phaux/rdza)
 - Italian: [ruggine](https://github.com/DamianX/ruggine)
-- Russian: [ржавчина](https://github.com/FluxIndustries/rzhavchina)
+- Russian: [Ржавый]([https://github.com/FluxIndustries/rzhavchina](https://github.com/Sanceilaks/rzhavchina))
```
Comment (bnjbvr) [inline README.md]:
```suggestion
- Russian: [Ржавый](https://github.com/Sanceilaks/rzhavchina)
```

**Pull request merged**